<?xml version="1.0"?>
<launch>
  
  <!-- args -->
  <arg name="from_database" default="False"/>
  <arg name="warehouse" default="DKN"/>
  <arg name = "path_unit" default="0.5"/>
  <arg name = "method" default="multi_json"/>
  
  <include file="$(find init_sim)/launch/empty_world_complete.launch">
    <arg name="world_name" value="$(find init_sim)/worlds/empty_world.world"/> 
    
    <arg name="paused" value="false"/>
    <arg name="use_sim_time" value="true"/>
    <arg name="gui" value="true"/>
    <arg name="recording" value="false"/>
    <arg name="debug" value="false"/>
  </include>

  <!-- <node pkg="init_sim" type="spawn_multiple_robots_1.py" name="spawn_multiple_robots_1" output="screen"/> -->

  <!-- <node pkg="init_sim" type="spawn_obstacles.py" name="spawn_obstacles" output="screen"/>
  <node pkg="init_sim" type="spawn_multiple_robots.py" name="spawn_multiple_robots" output="screen"/> -->

  <node pkg="init_sim" type="spawn_map.py" name="spawn_map" output="screen" args="=-from_database $(arg from_database) -warehouse $(arg warehouse) -path_unit $(arg path_unit) -method $(arg method)">
  </node>

</launch>
